Dockerfile

```diff
@@ -18,6 +18,7 @@ FROM python:3.10.2-slim-bullseye as base-image
 
 # Update system packages
 COPY scripts/install-base-packages.sh .
+COPY scripts/start-api.sh /start-api.sh
```

Comment:
Nit: I generally do this only in the runtime image because the other images don't need it and that maximizes the amount of Docker caching you can get when iterating on the start script (not that this is likely that frequent).